astroquery/alma/core.py

```diff
@@ -207,6 +208,92 @@ def _gen_sql(payload):
     return sql + where
 
 
+def get_enhanced_table(result):
+    """
+    Returns an enhanced table with quantities instead of values and regions for footprints.
+    Note that this function is dependent on the ``astropy`` - affiliated ``regions`` package.
+    """
+    try:
+        import regions
+    except ImportError:
+        print(
+            "Could not import astropy-regions, which is a requirement for get_enhanced_table function in alma"
+            "Please refer to http://astropy-regions.readthedocs.io/en/latest/installation.html for how to install it.")
+
+    def _parse_stcs_string(input):
+        csys = 'icrs'
+
+        def _get_region(tokens):
+            if tokens[0] == 'polygon':
+                if csys == tokens[1].lower():
+                    tokens = tokens[2:]
+                else:
+                    tokens = tokens[1:]
+                try:
+                    points = SkyCoord(
+                        [(float(tokens[ii]), float(tokens[ii + 1])) * u.deg
+                            for ii in range(0, len(tokens), 2)], frame=csys)
+                except Exception as ex:
+                    print(ex)
```

Comment:
this looks like debug code; we probably don't want to just catch and print exceptions here, right?  i.e., we'd rather raise this exception if there is a real bug?